.github/workflows/release.yml

```
@@ -76,14 +77,15 @@
           git config advice.ignoredHook false
           VERSION=${GITHUB_REF#refs/tags/v}
           git commit -m "chore(release): update CHANGELOG for v$VERSION [skip ci]" -a || echo "No changes to commit"
-          BRANCH=$(echo "${GITHUB_REF}" | sed 's@refs/tags/@@')
-          git push origin HEAD:$BRANCH || echo "Push to repository failed"
 
+          git checkout main || git checkout -b main 
+
+          git push origin main || echo "Push to repository failed"
 
       - name: Upload Release Assets
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)